src/store/modules/ADempiere/system.js

```diff
@@ -75,17 +91,20 @@ const system = {
       })
     }
   },
+
   getters: {
     getCountry: (state) => {
       return state.country
     },
-    getCurrency: (state) => {
-      const { currencyIsoCode, standardPrecision } = state.systemDefinition
+    getCurrency: (state, getter) => {
+      const { currencyIsoCode } = state.systemDefinition
 
-      return {
-        standardPrecision: standardPrecision || 2,
-        iSOCode: currencyIsoCode || 'USD'
-      }
+      return currencyIsoCode || 'USD'
+    },
+    getStandardPrecision: (state) => {
```

Comment:
Don't touch it if not exists changes

Reply:
The change was reverted, although the getter of getCurrency (which returns the standard precision and the currency code) is not used, only new functionalities were added.